ui/analyse/src/ctrl.ts

```diff
@@ -780,10 +780,10 @@ export default class AnalyseCtrl {
   }
 
   toggleComputer = () => {
+    if (this.showComputer() && this.opts.practice) this.togglePractice();
     if (this.ceval.enabled()) this.toggleCeval();
     const value = !this.showComputer();
     this.showComputer(value);
-    if (!value && this.practice) this.togglePractice();
```

Comment:
Now that `toggleCeval` turns off practice (when turning off computer evals), we want to `togglePractice` _first_ so that `toggleCeval` doesn't turn off practice only for `togglePractice` to turn it back on.

And if `togglePractice` turns practice on then `ceval` is enabled so `toggleCeval` doesn't turn practice off instantly